tensorflow/compiler/mlir/tosa/transforms/legalize_common.h

```diff
@@ -244,6 +244,12 @@ llvm::Optional<Value> convertFakeQuantOp(PatternRewriter& rewriter,
                                          double max, int64_t num_bits,
                                          bool narrow_range);
 
+llvm::Optional<Value> convertMirrorPadCommon(PatternRewriter& rewriter,
+                                             Operation* op,
+                                             RankedTensorType output_type,
+                                             Value input, Value pad,
+                                             StringRef mode);
```

Comment:
Lets make this an enum here and guard the function even be called with an invalid mode before we get to it.

Reply:
Define a common enum that aligns with string mode in TF and enum mode in TFL.